src/BL/Services/IMinioHelper.cs

```diff
@@ -28,5 +29,8 @@ public interface IMinioHelper
         Task<bool> BucketPolicySetPublic(string bucketName);
 
         Task DeleteObject(string bucketName, string objectKey);
+        Task<MinIOOperationResult> CreateUserWithMcAsync(string accessKey, string secretKey);
+        Task<MinIOOperationResult> SetUserPolicyAsync(string accessKey, string policyName);
+        Task<MinIOOperationResult> RemoveUserWithMcAsync(string accessKey);
```

Comment:
```suggestion
        Task<MinIOOperationResult> RemoveUserAsync(string accessKey);
```

As above, `WithMc` is an implementation detail